lib/g5kchecks/ohai/gpu.rb

```diff
@@ -4,39 +4,38 @@
 #gpu_output = "05:00.0 3D controller: NVIDIA Corporation GK210GL [Tesla K80] (rev a1)\n06:00.0 3D controller: NVIDIA Corporation GK210GL [Tesla K80] (rev a1)\n"
 #gpu_output = "03:00.0 3D controller: NVIDIA Corporation Device 17fd (rev a1)"
 
-if gpu_output != ''
-  gpu_lines = gpu_output.split("\n")
-  gpu_count = 0
-  gpu_vendor = 'Unknown'
-  gpu_model = 'Unknown'
+if gpu_output['NVIDIA']
+  nvidia_output = `nvidia-smi -L`
+  gpu_info_str = nvidia_output.split("\n")[-1].split()
+  gpu_model = gpu_info_str[3]
+  gpu_count = gpu_info_str[1].sub ':', ''
+  gpu_count = gpu_count.to_i + 1
 
-  gpu_lines.each { |line|
-    begin
-      gpu_model_info = line[/\[.*\]/]
-      gpu_model_info['['] = ''
-      gpu_model_info[']'] = ''
+  if gpu_model.include? 'P100'
```

Comment:
What else is on this line besides `P100`? Is none of it relevant?

Reply:
It's either a P100-SXM2-16GB  or P100-PCIE-16GB. The extra information doesn't look relevant to how we currently structure the grid5000 API. Although, we can always change that.

Reply:
This is now in the example output added to the file

Reply:
Great!